server/routes/varyingLicences/handlers/viewActiveLicence.ts

```diff
@@ -15,13 +17,20 @@ export default class ViewActiveLicenceRoutes {
 
     const shouldShowVaryButton = [LicenceStatus.ACTIVE].includes(<LicenceStatus>licence.statusCode)
 
-    const { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
+    let { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
       licence.additionalLicenceConditions
     )
 
+    if (isInPssPeriod) {
+      ;({ conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
+        (await this.licenceService.getParentLicenceOrSelf(licence.id.toString(), user))?.additionalLicenceConditions
+      ))
+    }
```

Comment:
as you're doing this twice and it's a bit complicated - maybe better to push this down in to the condition service and give it a nice name?